src/app/shared/config.ts

```diff
@@ -93,7 +93,7 @@ function getBaseUrl() {
     if (config.host === 'mail.ctemplarpizuduxk3fkwrieizstx33kg5chlvrh37nz73pv5smsvl6ad.onion') {
       return '/api/';
     }
-    return config.protocol + '//' + config.host.replace('mail.', 'api.') + '/';
+    return 'https://api.ctemplar.com';
```

Comment:
@notramo this can't be merged because it will break the tor support.
We have a different url for tor, so client opened in TOR browser should request tor api urls.

Reply:
There is no standard way to detect if site is hosted on IPFS, because there are many ways to bridge IPFS to HTTP (running bridge locally, or using a custom self-hosted bridge from another machine, or choosing from the many public bridges).
Can we detect if site is served on Tor? If there is a way to detect, then we should test it first, and if it is served on Tor, then return the onion address, if not, continue to this fallback. (If I understood the code correctly, Tor checking is done in line 93 before my code, and if it is on Tor, then the function returns at line 94 with the onion API address, and not even reaching my code.)